### PR TITLE
add italy anac spider

### LINKED ISSUE #867: ANAC Italy
**Links to bulk download and/or API endpoints**
https://dati.anticorruzione.it/opendata/organization/anticorruzione

**Link to CRM issue**
[CRM-7621](https://crm.open-contracting.org/issues/7621)

**Priority**
this week

### PR DESCRIPTION
**Pull request by Ravf95**
Closes #867

### CODE REVIEW

**Review by yolile — CHANGES_REQUESTED** [review]
This is a CKAN site, therefore you can do something similar to what is done in `argentina_buenos_aires`, `costa_rica_poder_judicial_records`, and others (see more at #404). I see that we can use https://dati.anticorruzione.it/opendata/api/3/action/package_search?q=ocds to list all of the OCDS related resources.

**Comment by Ravf95** [comment]
Thanks for the suggestion. I changed the base class to request that endpoint.

File: kingfisher_scrapy/spiders/italy_anac.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import scrapy
+
+from kingfisher_scrapy.base_spider import SimpleSpider
+from kingfisher_scrapy.util import components, handle_http_error
+
+
+class ItalyANAC(SimpleSpider):
+    """
+    Domain
+       Italy ANAC
```
Comment (yolile) [inline kingfisher_scrapy/spiders/italy_anac.py:10]:
```suggestion
       Autorità Nazionale Anticorruzione
```

File: kingfisher_scrapy/spiders/italy_anac.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import scrapy
+
+from kingfisher_scrapy.base_spider import SimpleSpider
+from kingfisher_scrapy.util import components, handle_http_error
+
+
+class ItalyANAC(SimpleSpider):
+    """
+    Domain
+       Autorità Nazionale Anticorruzione
```
Comment (yolile) [inline kingfisher_scrapy/spiders/italy_anac.py:10]:
```suggestion
       Autorità Nazionale Anticorruzione (ANAC)
```

**Review by yolile — APPROVED** [review]

**Pull request merged**